tests/components/steam_online/__init__.py

```diff
@@ -62,16 +67,32 @@ def ISteamUser(self) -> None:
 
     def GetFriendList(self, steamid: str) -> dict:
         """Get friend list."""
-        return {"friendslist": {"friends": [{"steamid": ACCOUNT_2}]}}
-
-    def GetPlayerSummaries(self, steamids: str) -> dict:
+        fake_friends = [{"steamid": ACCOUNT_2}]
+        for _i in range(0, 400):
```

Comment:
This makes the tests super heavy. Better to patch the “IDs per request” constant to 1 or 2.